Sources/SwiftProcessing/Core/UIKitElements/Capture.swift

```diff
@@ -0,0 +1,310 @@
+import UIKit
+import AVFoundation
+import Vision
+
+open class Camera: UIKitViewElement {
+    
+    private var previewView: UIView?
+    private(set) var cameraIsReadyToUse = false
+    
+    private let session = AVCaptureSession()
+    private weak var previewLayer: AVCaptureVideoPreviewLayer?
+    private lazy var sequenceHandler = VNSequenceRequestHandler()
+    private lazy var capturePhotoOutput = AVCapturePhotoOutput()
+    
+    
+    private lazy var dataOutputQueue = DispatchQueue(label: "FaceDetectionService",
+                                                     qos: .userInitiated, attributes: [],
+                                                     autoreleaseFrequency: .workItem)
+    
+    private var captureCompletionBlock: ((UIImage) -> Void)?
+    private var preparingCompletionHandler: ((Bool) -> Void)?
+    private var snapshotImageOrientation = UIImage.Orientation.upMirrored
+    
+    private var photo:Image?
+    
+    var AVCapturePositions = ["FRONT" : AVCaptureDevice.Position.front,
+                     "BACK" : AVCaptureDevice.Position.back]
+    
+    var AVCaptureQuality = ["HIGH" : AVCaptureSession.Preset.high,
+                            "MEDIUM" : AVCaptureSession.Preset.medium,
+                            "LOW" : AVCaptureSession.Preset.low,
+                            "VGA" : AVCaptureSession.Preset.vga640x480,
+                            "720" : AVCaptureSession.Preset.hd1280x720,
+                            "1080" : AVCaptureSession.Preset.hd1920x1080]
+    
+    init(_ view: Sketch) {
+        self.previewView = UIView()
+        super.init(view, self.previewView!)
+    }
+    
+    open func setResolution(resolution: String) {
```

Comment:
add the possible resolutions as constants to the main Sketch class. That way setResolution could be called like so

setResolution(HIGH)